scripts/test_fundraising/sale.py

```diff
@@ -0,0 +1,62 @@
+import json
+import subprocess
+import datetime
+import time
+
+valAddress = 'spnvaloper1ezptsm3npn54qx9vvpah4nymre59ykr9mx22g4'
```

Comment:
Also FYI, variable names  and functions in python should not use camelcase, but include '_' to separate words ([PEP8: function and variable names](https://peps.python.org/pep-0008/#function-and-variable-names)). We are not using python as our main language but we may want to follow standards as much as possible since we are writing OSS.